tools/proofers/for_mentors.php

```diff
@@ -208,16 +208,17 @@ function output_project_details($mentored_round, $projectid, $nameofwork, $autho
 
 function get_beginner_projects_in_state($state)
 {
-    $sql = "
+    $sql = sprintf("
         SELECT projectid, nameofwork, authorsname
         FROM projects
         WHERE
             difficulty = 'BEGINNER' AND
-            state='$state'
+            state='%s'
         ORDER BY
-            modifieddate ASC
-    ";
-    $result = mysqli_query(DPDatabase::get_connection(), $sql);
+            modifieddate ASC",
+        DPDatabase::escape($state)
```

Comment:
we didn't encode before, we're sure all callers are not encoding already and we won't introduce double encoding, right?

Reply:
True, but there are only 2 callers and they're just using one of the `Round::foo_state` fields which will be something like `P3.avail`.

```$ ag get_beginner_projects_in_state
tools/proofers/for_mentors.php
123:$projects_available = get_beginner_projects_in_state($mentoring_round->project_available_state);
147:$projects_waiting = get_beginner_projects_in_state($mentoring_round->project_waiting_state);
209:function get_beginner_projects_in_state($state)
```

We don't really need to escape, but I figured I may as well do it for consistency. I can undo that if you like.